Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "payable-asset-smart-contract"
-version = "0.1.1"
+version = "0.2.0"
```

Comment:
Migrations are now version-stamped, so it's important we bump the version here before we release new code for migraiton